apps/emqx/src/emqx_os_mon.erl

```diff
@@ -131,7 +132,7 @@ handle_info({timeout, _Timer, mem_check}, #{sysmem_high_watermark := HWM} = Stat
 handle_info({timeout, _Timer, cpu_check}, State) ->
     CPUHighWatermark = emqx:get_config([sysmon, os, cpu_high_watermark]) * 100,
     CPULowWatermark = emqx:get_config([sysmon, os, cpu_low_watermark]) * 100,
-    CPUVal = emqx_vm:cpu_util(),
+    CPUVal = cpu_sup:util(),
```

Comment:
why this change? for accuracy?

Reply:
`cpu_sup:util/0` will be called in single gen_server `emqx_os_mon`, no need to warp it in `emqx_cpu_sup_worker` process.